apollo-router/src/apollo_studio_interop/mod.rs

```diff
@@ -230,6 +231,7 @@ impl ComparableUsageReporting {
 /// Generate a ComparableUsageReporting containing the stats_report_key (a normalized version of the operation signature)
 /// and referenced fields of an operation. The document used to generate the signature and for the references can be
 /// different to handle cases where the operation has been filtered, but we want to keep the same signature.
+#[tracing::instrument(skip_all, level = "debug")]
```

Comment:
are all the tracing annotations still needed?

Reply:
each of those annotations creates a new span, and there's some overhead for each created span even if not at a visible level

Reply:
I thought they would be useful information to have in the trace view, but can remove if it's a performance issue. Are the spans created even if you're not running the router with `--log=trace`?